compiler/rustc_mir_transform/src/pass_manager.rs

```diff
@@ -205,6 +229,7 @@ fn run_passes_inner<'tcx>(
     passes: &[&dyn MirPass<'tcx>],
     phase_change: Option<MirPhase>,
     validate_each: bool,
+    no_opt: bool,
```

Comment:
Could you change `no_opt` everywhere to `no_optimize` or `do_not_optimize`? I keep getting confused between **opt**ion and **opt**imization 😅 

And/or potentially also use an enum `Optimizations { Enable, Disable }` or something) instead of a `bool`.